### DISCUSSION TITLE
Addition of Performance profile and Performance Mode

### DISCUSSION THREAD

**Discussion by monsiu** [salvogiangri/UN1CA#137] [Feature requests]
The flagships have these modes and it would be cool to have them in the ROM.

**Comment by salvogiangri** [comment]
Unfortunately this feature cannot be implemented on every device as it requires proper hardware support, more specifically it will only work properly for the following family of SoCs (availability may vary from device to device):
- Snapdragon 865 (SM8250)
- Snapdragon 888 (SM8350)
- Snapdragon 8 Gen 1 (SM8450)
- Snapdragon 8+ Gen 1 (SM8475)
- Snapdragon 8 Gen 2 (SM8550)
- Snapdragon 8 Gen 3 (SM8650)
- Exynos 9820/9825 (Makalu)
- Exynos 990
- Exynos 2100
- Exynos 2200 (S5e9925)
- Exynos 2400 (S5e9945)

**Reply by monsiu** [reply]
> Unfortunately this feature cannot be implemented on every device as it requires proper hardware support, more specifically it will only work properly for the following family of SoCs (availability may vary from device to device):
> 
>     * Snapdragon 865 (SM8250)
> 
>     * Snapdragon 888 (SM8350)
> 
>     * Snapdragon 8 Gen 1 (SM8450)
> 
>     * Snapdragon 8+ Gen 1 (SM8475)
> 
>     * Snapdragon 8 Gen 2 (SM8550)
> 
>     * Snapdragon 8 Gen 3 (SM8650)
> 
>     * Exynos 9820/9825 (Makalu)
> 
>     * Exynos 990
> 
>     * Exynos 2100
> 
>     * Exynos 2200 (S5e9925)
> 
>     * Exynos 2400 (S5e9945)

OOF! Thanks for the clarification though. I see the a52 has Snapdragon 720G so im outta luck haha.

**Reply by HammoudAbbas** [reply]
> Unfortunately this feature cannot be implemented on every device as it requires proper hardware support, more specifically it will only work properly for the following family of SoCs (availability may vary from device to device):
> 
> * Snapdragon 865 (SM8250)
> * Snapdragon 888 (SM8350)
> * Snapdragon 8 Gen 1 (SM8450)
> * Snapdragon 8+ Gen 1 (SM8475)
> * Snapdragon 8 Gen 2 (SM8550)
> * Snapdragon 8 Gen 3 (SM8650)
> * Exynos 9820/9825 (Makalu)
> * Exynos 990
> * Exynos 2100
> * Exynos 2200 (S5e9925)
> * Exynos 2400 (S5e9945)

Which i guess it means that it can be implemented to s20 fe

**Reply by salvogiangri** [reply]
> Which i guess it means that it can be implemented to s20 fe

S20 FE already has this enabled.

**Reply by HammoudAbbas** [reply]
I knew I'd look like an idiot 🥲💔
